l10n_it_pos_fatturapa/static/src/js/screens.js

```diff
@@ -0,0 +1,20 @@
+odoo.define('l10n_it_pos_fatturapa.screens', function (require) {
+    "use strict";
+
+    var Screens = require('point_of_sale.screens');
+
+    Screens.ClientListScreenWidget.include({
+        display_client_details: function(visibility,partner,clickpos) {
```

Comment:
@robyf70 questo a cosa serve? Puoi spiegarlo in un commento?

Reply:
@eLBati Estende il metodo di display per aggiungere l'azione sul checkbox per gestire il valore ``true|false`` sul ``value`` del nodo associato al checkbox. Inoltre se il nodo è ``electronic_invoice_subjected`` nascondo o visualizza il blocco di campi per la fattura elettronica